HarmonyCore/FileIO/FileChannelManager.dbl

```diff
@@ -156,19 +456,23 @@ namespace Harmony.Core.FileIO
             fileName, @string
         proc
             data closedChannelCount = 0
-            data openDictionary, @ConcurrentDictionary<FileOpenMode, ConcurrentBag<int>>
-            if(mChannelLookup.TryGetValue(fileName, openDictionary))
-            begin
-                data openDictionaryItem, @KeyValuePair<FileOpenMode, ConcurrentBag<int>>
-                foreach openDictionaryItem in openDictionary
+            data fileNameLookup, @ConcurrentDictionary<string, ConcurrentDictionary<FileOpenMode, ConcurrentBag<int>>>
+            if (mChannelLookup.TryGetValue(RemoteFile(fileName) ? LoadInfo.CurrentLoadId : 0, fileNameLookup))
```

Comment:
Can you skip the call to RemoteFile if we're not doing any of this load balancing?

Reply:
Check in the new commit whether that would work. Thanks.